config/manager/usde-config.yaml

```diff
@@ -47,11 +52,21 @@ data:
       includeSubfields: true
     - path: spec.vertices.udf.builtin
       includeSubfields: true
-    - path: spec.sideInputs
+    - path: spec.sideInputs.container.image
+    - path: spec.sideInputs.container.command
+    - path: spec.sideInputs.container.args
+    - path: spec.sideInputs.container.env
```

Comment:
Can you also update the e2e test files related to this:
- https://github.com/numaproj/numaplane/blob/main/tests/manifests/special-cases/ppnd/usde-config.yaml
- https://github.com/numaproj/numaplane/blob/main/tests/manifests/special-cases/progressive/usde-config.yaml

Reply:
thanks for catching! I already forgot about that even though we were just discussing it. :) It sounds like a good motivation for moving the `defaultUpgradeStrategy` out!

Reply:
Working on that now, should be ready soon